Assets/Scripts/Core/GameController.cs

```diff
@@ -19,17 +19,17 @@ public class GameController : MonoBehaviour {
 		public delegate void MoveEvent(Vector3 position);
 		public event MoveEvent Moved;
 
-		private Grid<Tilemap.Node> grid_;
-		private Tilemap tilemap_;
+		public Grid<Tilemap.Node> grid {get; private set;}
+
 		public GridPathfinding gridPathfinding { get; private set; }
+		public Tilemap tilemap {get; private set;}
```

Comment:
Inconsistent with the spacing after `{ get;`

Reply:
Already fixed.